purchase_report_date_planned/models/purchase_report.py

```diff
@@ -0,0 +1,22 @@
+from odoo import fields, models
+
+
+class PurchaseReport(models.Model):
+    _inherit = "purchase.report"
+
+    date_planned = fields.Datetime(compute="_compute_date_planned", store=True)
```

Comment:
you do not have to create a computed field. it's just a normal field and as the model is based on a sql view the data will be here (with the override of _select and _group_by)